### ISSUE TITLE
is there anyone who got the eao value like the author?

### ISSUE THREAD

**Issue by yjkupup** [foolwood/SiamMask#179]
i follow the steps like readme shows, and use the same environment with author's. but still can't get the eao value 0.38.  the eao on vot2018 i get is 0.34, still far from 0.38. so is there anyone who got 0.38 or very close to it ? is there any tricks the code doesn't show? thanks very much!!!

**Comment by zosel260** [comment]
@yjkupup 
I tried training the base and  refinement module twice. But the final EAOs were only 0.350 and 0.356  on VOT2018.
I think the only different things from author's were batch size and the randomly applied augmentation process.
I set the batch size like this:
1st training:  base module (64->16),  refinement module(32)
2nd training: base module (48->32), refinement module(64)
( at epoch 10 of base module, I got out of memory problem, I reduced batch size)

How did you set the  batch size ?  Did you set it with 64 for both base & refinement problem?

**Comment by yjkupup** [comment]
@zosel260 I keep the batchsize of 64 when training base and refine model, the only difference is the cuda version(mine is 10.2, the author's is  9.2).  The final eao is only 0.342, it's far away from the author's. So i don't know what is problem. The result is not satisfying at all.
I also encountered the problem of memory. It showed the server doesn't have enough memory, the process is killed. Then i added 300G virtual memory and it worked. 
My training log includes a warning: UserWarning: nn.Upsample is deprecated. Use nn.functional.interpolate instead. So, do you think it is caused by the discordance of cuda version?  What is your cuda version?

**Comment by zosel260** [comment]
@yjkupup  My cuda version is 11.0. I'm not sure nn.Upsample is problem but I don't think it has critical effect.
Did you test other data set like VOT2019, I tested author's models on it. the result were poor like this below.

```
------------------------------------------------------------------------------------
|            Tracker Name            | Accuracy | Robustness | Lost Number |  EAO  |
------------------------------------------------------------------------------------
| Custom_mask_refine_SiamMask_VOT_LD |  0.596   |   0.456    |    91.0     | 0.289 |
|  Custom_mask_refine_SiamMask_VOT   |  0.609   |   0.517    |    103.0    | 0.280 |
------------------------------------------------------------------------------------
```
 I know the tracking parameters(penalty_k, window influence, learning rate ) were optimized only for VOT2018,
 but the VOT2019's results were quite low than my expectation.  I feel the performance is quite sensitive to dataset.
So maybe the augmentation process (blur, gray, shift) during training could affects the result.

By the way,  I use  one local GPU(Titan V). Can I add virtual memory for GPU? If yes, could you give me any tips to add it?

**Comment by yjkupup** [comment]
@zosel260  I use four 1080ti for training. The memory problem i encountered was cpu memory running-out. So adding virtual memory worked. We can't add virtual memory for gpu. 
I havn't tested  on vot2019.

**Comment by zosel260** [comment]
@yjkupup  thanks!

**Comment by zosel260** [comment]
@yjkupup 
Did you check the evaluation functions( utils/pysot/evaluation/ eao_benchmark.py)  are  proper tools for VOT metric?

**Comment by yjkupup** [comment]
@zosel260 Yes, i checked it, it's proper. But did you find that there are some details in github code are inconsistent with the paper?such as the learning rate for the epoch 5-20, and the weights of mask/loc/reg loss.
